### ISSUE TITLE
Malformed parsing for non quoted attributes.

### ISSUE THREAD

**Issue by jasonhilder** [kkmtyyz/parsercher#2]
In instances where attributes are not within quotes it breaks the parsing.

For example the *src* attribute:
```
<img class="img-fluid" src=https://www.example.com/banner-1.jpg >
```

I understand this is invalid html, but is quite common on the web to have quoteless data attributes.

**Comment by kkmtyyz** [comment]
Thank you for trying parsercher.
I also appreciate the information provided.
I have released a modified v3.1.5.
I've tested it, but if it doesn't work, please let me know again.

github: https://github.com/kkmtyyz/parsercher/releases/tag/v3.1.5
crates.io: https://crates.io/crates/parsercher/3.1.5

**Comment by JSONhilder** [comment]
Thanks so much for the quick fix much appreciated.
